fluentd/datadog_checks/fluentd/fluentd.py

```diff
@@ -15,6 +19,7 @@ class Fluentd(AgentCheck):
     SERVICE_CHECK_NAME = 'fluentd.is_ok'
     GAUGES = ['retry_count', 'buffer_total_queued_size', 'buffer_queue_length']
     _AVAILABLE_TAGS = frozenset(['plugin_id', 'type'])
+    VERSION_PATTERN = r'.* (?P<version>[0-9\.]+)$'
```

Comment:
(nit) The ending `$` is probably not needed. Removing it will make the integration still work, if something is added after later.

```suggestion
    VERSION_PATTERN = r'.* (?P<version>[0-9\.]+)'
```